packages/core/src/payment/strategies/bolt/bolt-payment-strategy.ts

```diff
@@ -351,4 +356,23 @@ export default class BoltPaymentStrategy implements PaymentStrategy {
             throw new PaymentArgumentInvalidError();
         }
     }
+
+    private _setExtraItemsForAnalytics() {
```

Comment:
Since we already have an access modifier, we can omit underscore

```suggestion
    private setExtraItemsForAnalytics() {
```